### PR TITLE
redirect to saved url

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by alemata** [comment]
@flavaiogranero

File: 01-Login/app/auth.service.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ declare var Auth0Lock: any;
 @Injectable()
 export class Auth {
   // Configure Auth0
-  lock = new Auth0Lock('YOUR_CLIENT_ID', 'YOUR_DOMAIN', {});
+  lock = new Auth0Lock('oCzezT18YMr3HjyZlYqIRUFgbjAKNU6J', 'amata2.auth0.com', {});
```
Comment (flaviogranero) [inline 01-Login/app/auth.service.ts]: @alemata I think you forgot this

File: 04-User-Profile/app/auth.service.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ declare var Auth0Lock: any;
 @Injectable()
 export class Auth {
   // Configure Auth0
-  lock = new Auth0Lock('YOUR_CLIENT_ID', 'YOUR_DOMAIN', {
+  lock = new Auth0Lock('oCzezT18YMr3HjyZlYqIRUFgbjAKNU6J', 'amata2.auth0.com', {
```
Comment (flaviogranero) [inline 04-User-Profile/app/auth.service.ts]: here also

File: 04-User-Profile/app/profile_edit.component.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ export class ProfileEdit {
     });
 
     this.authHttp
-      .patch('https://' + 'YOUR_DOMAIN' + '/api/v2/users/' + this.auth.userProfile.user_id, data, {headers: headers})
+      .patch('https://' + 'amata2.auth0.com' + '/api/v2/users/' + this.auth.userProfile.user_id, data, {headers: headers})
```
Comment (flaviogranero) [inline 04-User-Profile/app/profile_edit.component.ts]: and here

File: 01-Login/app/auth.service.ts (hunk 1 above)
Comment (alemata) [inline 01-Login/app/auth.service.ts]: strangeee!

**Comment by alemata** [comment]
@flaviogranero updated! sorry

**Comment by flaviogranero** [comment]
@chenkie 👍

**Pull request merged**